src/containers/LoginFormContainer.js

```diff
@@ -1,114 +1,69 @@
 import React, { Component } from "react";
-import FacebookLogin from "react-facebook-login";
-import GoogleLogin from "react-google-login";
-import { BASE_URL, API_FB_ID, API_GOOGLE_ID } from "../constants";
-import { providers, roles } from "../enums.js";
-import axios from "axios";
-import { Redirect } from "react-router-dom";
-import HomePage from "../components/HomePage/HomePage";
-import EditProfileContainer from "./EditProfileContainer";
+import { API_FB_ID, FACEBOOK_SCOPES, API_GOOGLE_ID } from "../constants";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import {
+  loginRequested,
+  handleGoogleSuccess,
+  handleGoogleFailure,
+  handleFacebookResponse
+} from "../actions/LoginActions";
```

Comment:
you can use:
**import * as loginActions from "../actions/LoginActions";** and then in mapDispatchToProps:
`const mapDispatchToProps = dispatch => 
 bindActionCreators(
    {   ...loginActions
    },
    dispatch
  );`

Reply:
Agreed